agent/api/task/task_linux.go

```diff
@@ -217,3 +218,25 @@ func (task *Task) dockerCPUShares(containerCPU uint) int64 {
 	}
 	return int64(containerCPU)
 }
+
+// requiresCredentialSpecResource returns true if at least one container in the task
+// needs a valid credentialspec resource
+func (task *Task) requiresCredentialSpecResource() bool {
+	return false
+}
```

Comment:
I understand that this is a windows-specific change; I'd like to see a linux-specific comment.  I can see methods like this (and those below it) being confusing in future to those who don't have the context of this pr.

Reply:
@vsiddharth this sounds like a good idea. consider to open a pr to add a bit context into it.